src/components/index.js

```diff
@@ -8,6 +8,7 @@ export {default as Wedding} from "./Wedding";
 export {default as Travel} from "./Travel";
 export {default as Event} from "./Event";
 export {default as EventImage} from "./EventImage";
-export {default as Questionnaire} from "./Questionnaire";
+export {default as Email} from "./Email";
 export {default as FAQ} from "./FAQ";
```

Comment:
I'd be consistent about file names here too `FAQ` should probably be `Faq`?

Reply:
Was thinking that since FAQ is an acronym same as COVID - will keep it all capitalized.